temporalio/common.py

```diff
@@ -1063,6 +1065,65 @@ def from_canonical_string(canonical: str) -> WorkerDeploymentVersion:
             )
         return WorkerDeploymentVersion(parts[0], parts[1])
 
+    def _to_proto(self) -> temporalio.api.deployment.v1.WorkerDeploymentVersion:
+        """Convert to proto representation."""
+        return temporalio.api.deployment.v1.WorkerDeploymentVersion(
+            build_id=self.build_id,
+            deployment_name=self.deployment_name,
+        )
+
+
+class VersioningOverride(ABC):
```

Comment:
I assume this may be set on child workflow or activity invocations one day and is why it is in `common`?

Reply:
Yes